### PR TITLE
Update the Kusto extension to the insiders market place

### PR DESCRIPTION
**Pull request by joberume**
Add the entry for the Kusto extension to the insiders marketplace. Will add to main market place later...

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -3651,6 +3651,59 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "74",
+					"extensionName": "kusto",
+					"displayName": "Kusto",
```
Comment (qifahs) [inline extensionsGallery-insider.json:3658]: The display name should be "Kusto (KQL)"

File: extensionsGallery-insider.json [hunk 2]
```diff
@@ -3651,6 +3651,59 @@
...
+					"shortDescription": "Kusto extension for Azure Data Studio",
```
Comment (qifahs) [inline extensionsGallery-insider.json:3659]: Same here "Kusto (KQL) extension for...

File: extensionsGallery-insider.json [hunk 3]
```diff
@@ -3651,6 +3651,59 @@
...
+									"source": "https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/kusto/package.json"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.License",
+									"source": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt"
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.Engine",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:3695]: `"key": "Microsoft.AzDataEngine"` as well

File: extensionsGallery-insider.json [hunk 4]
```diff
@@ -3651,6 +3651,63 @@
...
+									"value": ">=1.22.0"
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json:3696]: here you could simply use *, this is to specify the vscode version not ADS version.

**Pull request merged**